README.md

```diff
@@ -293,7 +293,8 @@ An X server running on Windows is required for running Linux GUI apps on Windows
 
 - [Bash](https://www.gnu.org/software/bash/) - Bash is the GNU Project's shell. Bash is the Bourne Again SHell. Bash is an sh-compatible shell that incorporates useful features from the Korn shell (ksh) and C shell (csh).
 - [Cygwin](https://cygwin.com) - Cygwin is a Unix-like environment and command-line interface for Microsoft Windows.
-- [Babun](http://babun.github.io) - A Linux-like console on a Windows host. ![github project][githublogo]
+- [Cmder](https://cmder.net/) - A very nice console emulator built on ConEmu. ![github project][githublogo]
+- [Babun](http://babun.github.io) - [PROJECT DISCONTINUED] A Linux-like console on a Windows host. ![github project][githublogo]
```

Comment:
Should Babun be removed all together?

Reply:
Yes, I think so. Awesome lists are supposed to be the best of the best.